iina/MPVController.swift

```diff
@@ -1025,7 +1025,7 @@ class MPVController: NSObject {
             // video is paused to avoid wasting energy with needless processing. If paused shutdown
             // the timer that synchronizes the UI and the high priority display link thread.
             if paused {
-              player.invalidateTimer()
+              if ![.willShow, .shown].contains(player.mainWindow.animationState) { player.invalidateTimer() }
```

Comment:
This fixes the problem, but is undesirable from an energy use perspective. We absolutely do not want the timer to run when playback is paused and therefore the video position is not changing.